docs/overview.md

```diff
@@ -48,40 +48,27 @@ technologies involved and be able to set up the basic stuff yourself.
 As a to-do list, before you can even install Ansible-NAS, you'll have to:
 
 1. Choose, buy, configure, and test your own **hardware**. Note that ZFS loves
-   RAM - it will run [with 1 GB](https://wiki.freebsd.org/ZFSTuningGuide), but
-   it won't be happy. The ZFS on Linux (ZoL) people
-   [recommend](https://github.com/zfsonlinux/zfs/wiki/FAQ#hardware-requirements)
-   at least 8 GB for best performance, but the more, the better. As robust as
-   ZFS is, it assumes the data in memory is correct, so [very bad
-   things](http://research.cs.wisc.edu/adsl/Publications/zfs-corruption-fast10.pdf)
-   happen to your data if there is memory corruption. For this reason, it is
-   [strongly
-   recommended](https://github.com/zfsonlinux/zfs/wiki/FAQ#do-i-have-to-use-ecc-memory-for-zfs)
-   to use ECC RAM. ZFS also prefers to have the hard drives all to itself. If
-   you're paranoid (a good mindset when dealing with servers), you'll probably
-   want an uninterruptible power supply (UPS) of some sort as well and SMART
-   monitoring for your hard drives. See the [FreeNAS hardware
-   requirements](https://freenas.org/hardware-requirements/) as a guideline.
+   RAM, and it is [recommended](zfs_overview.md) you use ECC RAM. ZFS also
+   prefers to have the hard drives all to itself. If you're paranoid (a good
+   mindset when dealing with servers), you'll probably want an uninterruptible
+   power supply (UPS) of some sort as well and SMART monitoring for your hard
```

Comment:
Interested in what you mean by "as well and SMART monitoring for your hard drives"?

Reply:
I also think a home NAS project recommending ECC is way overkill. I run Ansible-NAS on an old HP Microserver. I don't do enough io that ECC memory is going to save me from any sort of data-loss.

Reply:
I've just read more of the docs and realised that you say what I say regarding ECC, so we should be consistent.

Reply:
I think a comment should be made that while ECC ram is best, make sure your processor and chipset support it. I bought ECC ram not knowing this and my i3 doesn't support ECC. Wasted $50 on memory I cannot use.

Reply:
Good point, will update.